uninstall.php

```diff
@@ -30,6 +30,8 @@
 	include_once dirname( __FILE__ ) . '/packages/woocommerce-admin/src/Install.php';
 	\Automattic\WooCommerce\Admin\Install::drop_tables();
 
+	define( 'WC_PLUGIN_BASENAME', plugin_basename( __FILE__ ) );
```

Comment:
This would set `WC_PLUGIN_BASENAME` to `woocommerce/uninstall.php` which is not correct. This might not be very important during the uninstall process, but it'd be best to set it to the correct value.

Keep in mind that `__FILE__` (in `uninstall.php`) refers to `uninstall.php` instead of `woocommerce.php` which is the file used to set this constant under normal circumstances (see [#](https://github.com/woocommerce/woocommerce/blob/bccc80366dd22f3fd2ca6b2eb21a8c3a95e157ca/plugins/woocommerce/woocommerce.php#L20), [##](https://github.com/woocommerce/woocommerce/blob/bccc80366dd22f3fd2ca6b2eb21a8c3a95e157ca/plugins/woocommerce/includes/class-woocommerce.php#L276)).